annotator/__main__.py

```diff
@@ -244,6 +246,16 @@ def _getAnnotationKey(dirs):
 
         for directory in dirs:
 
+            # try to circumvent a SSL certificate verification failure; not optimal but saves from 
+            # somewhat obscure errors; in this case we trust ourselves... with a warning
+            if (not os.environ.get('PYTHONHTTPSVERIFY', '') and getattr(ssl, '_create_unverified_context', None)): 
+                print()
+                print("WARNING: CERTIFICATE VERIFICATION FAILURE. This is probably benign and due to outdated openssl installation. Please upgrade your openssl package to the latest version (may require a pip/conda update).")
```

Comment:
Is there a way to detect the system openssl version and check whether it is compatible?

At a minimum I think these should be true warnings rather than print statements.

Reply:
:) these are lines of codes that I felt bad adding... the issue is that later versions of python by default perform certificate validation when libraries like urllib try to open https connections; there are different reasons for verification failure on mac os x vs. other systems (e.g. python 3.6 doesn't by default use mac os x root certificates and can't find them; the version of openssl shipped with python 3.7 might not work with verifying certificates coming from a package - e.g. certifi - that was compatible with an earlier version of openssl, and other fun things :)). If anyone figures out a way to detect these problems and suggest better resolution that would be great. I didn't have time to get through it in a more meaningful way; this warning message is a bit misleading, too and more of an eyesore to force us resolve that better.